### DISCUSSION TITLE
How can I stop Cyberduck from trying to connect to a no-longer existing Google Drive?

### DISCUSSION THREAD

**Discussion by seberkuhn** [iterate-ch/cyberduck#14441] [Q&A] [answered]
Ever since Google Drive changed where it is storing its information, Cyberduck keeps popping up the following error message: "There was a problem connecting to the server "127.0.0.1". This file server will not allow any additional users to log on. Try to connect again later". Followed by an "OK" button - which I have to click about a zillion times (the message keeps popping up again and again) until it goes away. I can't even quit Cyberduck while this alert is open.
I should say that I have absolutely no need for Cyberduck to connect to my Google drive, and it still connects to my other services as always. However, I cannot find any menu or preference item that would allow me to disable this futile attempt to connect to the Google drive which does not exist at that location anymore. Any help would be much appreciated!

**Comment by ylangisc** [comment, marked as answer]
It should help to delete all files inside the `Sessions` in the [Application Support folder](https://docs.cyberduck.io/cyberduck/support/#application-support-folder) after quitting Cyberduck.

**Reply by seberkuhn** [reply]
Thanks for the advice - I did delete the (one) file in that folder, but it had no effect on the problem I am encountering. All that happened is that Cyberduck didn't open any connections on startup like it would usually do (the last connections I used). Should I delete files in any of the other folders? (Profiles, History, Bookmarks,...)

**Comment by Cassie205** [comment]
In general, the error message you're experiencing only appears while reconnecting with leftovers from the previous connection, especially after a change in the login settings as you initially described. Deleting the _Sessions_ folder entries erases all information on previous connections via Cyberduck except the keychain entries. 

To verify that no connection to the old bookmark can be formed make sure to delete the bookmark for the connection as well as the corresponding keychain entries.

**Reply by seberkuhn** [reply]
Thank you. As I already pointed out earlier, deleting the "Sessions folder" content did NOT help. I didn't want to delete any bookmarks since it would be a lot of extra work to recreate them. I did check all of them to make sure that there was NOTHING in any of them pointing to the Google Drive.
Be this as it may, for no apparent reason whatsoever, Cyberduck finally stopped this behavior on its own. Now it works as always - no error messages. I consider this topic therefore as closed.
